oci.go

```diff
@@ -99,3 +99,18 @@ func processRunning(pid int) (bool, error) {
 
 	return true, nil
 }
+
+func updateStoppedContainer(podStatus vc.PodStatus) error {
```

Comment:
I know it is a bit confusing because the specification forces us to do something confusing here... It updates the container state in case we detected the process is not running anymore.